doc/Advanced-configuration.md

```diff
@@ -1,24 +1,32 @@
-For advanced configuration use the following files: `ejabberd.cfg`, `vm.args` and `app.config`. The first one contains all pure XMPP settings, the second affects Erlang VM behaviour (performance tuning, node name) and third usually has to be modified to change low-level logging parameters. Since you've gotten this far, we assume you're already familiar with Erlang syntax.
+For advanced configuration use the following files:
+
+* `ejabberd.cfg` for pure XMPP settings, 
```

Comment:
hmm... that was in the original note. I'll think about rephrasing it, any ideas of what meaning I should convey?